docs/configuration/vpn/ipsec.rst

```diff
@@ -13,10 +13,10 @@ address, which makes it easier to setup static routes or use dynamic routing
 protocols without having to modify IPsec policies. The other advantage is that
 it greatly simplifies router to router communication, which can be tricky with
 plain IPsec because the external outgoing address of the router usually doesn't
-match the IPsec policy of typical site-to-site setup and you need to add special
-configuration for it, or adjust the source address for outgoing traffic of your
-applications. GRE/IPsec has no such problem and is completely transparent for
-the applications.
+match the IPsec policy of a typical site-to-site setup and you would need to
+add special configuration for it, or adjust the source address for your
+applications' outgoing traffic. GRE/IPsec has no such problem and is completely
```

Comment:
Missplaced `'`?

Reply:
applications are plural, and it’s a possessive apostrophe
I can reword it to make it simpler if you’d prefer.

Reply:
Shouldn't it be `application's` then instead?

Reply:
The intended syntax here is `possessive(plural('application'))`. In different words, "or adjust the source address for outgoing traffic of your applications". I might prefer the latter, but I have no objections against "applications' outgoing traffic".

Reply:
Your wording here is clearer. I’ll update the wording to match, thanks!